composer.json

```diff
@@ -22,6 +22,6 @@
 		}
 	],
 	"autoload": {
-		"files": [ "src/UserAgentParser.php" ]
+		"files": [ "src/UserAgentParser.php", "src/browser_constants.php" ]
```

Comment:
I think it should use the `PSR-4` autoloader to load the related namespace automatically.

Reply:
PHP only triggers the auto loader lookup on missing classes. There are no classes here.

Reply:
To avoid function name collision, I think we can give the namespace for the `parse_user_agent` function.

We don't guarantee that this function name will be the same on other packages :).

Reply:
This library has existed since before namespaces were added to the language, and as noted in the description I do not wish to break compatibility at this time. Maybe in the future but currently I am bound to the global function.